src/gazebo_ros_link_attacher.cpp

```diff
@@ -253,6 +257,16 @@ namespace gazebo
       }
       detach_vector.clear();
     }
+
+    joint_state_ = sensor_msgs::JointState();
```

Comment:
are there any plans to use the `joint_state_` member somewhere else? otherwise, it could just be a local variable with the `if (joint_state_pub_.getNumSubscribers() > 0)` check at the beginning